ansible/playbooks/install-precice.yml

```diff
@@ -0,0 +1,191 @@
+---
+- name: Install preCICE and related tutorials
+  hosts: all
+  become: true
+  vars:
+    user_home: "/home/vagrant"
+    precice_repo: "https://github.com/precice/precice.git"
+    tutorials_repo: "https://github.com/precice/tutorials.git"
+    fortran_module_repo: "https://github.com/precice/fortran-module.git"
+    python_bindings_repo: "https://github.com/precice/python-bindings.git"
+    precice_path: "{{ user_home }}/precice"
+    venv_path: "{{ user_home }}/python-venvs/pyprecice"
+
+  tasks:
+    - name: Install preCICE build dependencies
+      apt:
+        name:
+          - cmake
+          - libeigen3-dev
+          - libxml2-dev
+          - libboost-all-dev
+          - petsc-dev
+          - python3-dev
+          - python3-numpy
+        state: present
+        update_cache: yes
+
+    - name: Clone preCICE repo if not present
+      git:
+        repo: "{{ precice_repo }}"
+        dest: "{{ precice_path }}"
+        version: main
+        depth: 1
+        update: yes
+      become: false
+
+    - name: Build and package preCICE
+      become: false
+      shell: |
+        mkdir -p build && cd build
+        cmake -DCMAKE_INSTALL_PREFIX=/usr -DBUILD_SHARED_LIBS=ON \
+              -DCMAKE_BUILD_TYPE=RelWithDebInfo \
+              -DPRECICE_RELEASE_WITH_DEBUG_LOG=ON -DBUILD_TESTING=OFF -Wno-dev ..
+        make -j $(nproc)
+        rm -fv ./*.deb && make package
+        sudo apt-get install -y ./libprecice*_*.deb
+        rm -rfv ./*.deb ./*.tar.gz _CPack_Packages
+      args:
+        chdir: "{{ precice_path }}"
+        executable: /bin/bash
+
+    - name: Copy examples from /usr/share
+      copy:
+        remote_src: true
+        src: /usr/share/precice/examples/
+        dest: "{{ precice_path }}-examples"
```

Comment:
This playbook currently fails at this point. The original is:

```
sudo apt-get install -y ./libprecice*_*.deb

...

cp -r /usr/share/precice/examples/ ./precice-examples
```

It looks like the `src` and `dest` are swapped here.